ansible_collections/arista/avd/roles/anta_runner/README.md

```diff
@@ -217,6 +217,10 @@ avd_catalogs_enabled: true
 # Generate tests for BGP peers in VRFs.
 avd_catalogs_allow_bgp_vrfs: false
 
+# Generate extra fabric-wide validation tests (e.g., reachability and routing tests).
```

Comment:
here would be a good place to properly list maybe? or having a very clear section